sql/parse/parse.go

```diff
@@ -502,6 +502,13 @@ func convertShow(ctx *sql.Context, s *sqlparser.Show, query string) (sql.Node, e
 
 		return node, nil
 	case "create procedure":
+		dbName := s.Table.Qualifier.String()
+		if dbName == "" {
+			dbName = ctx.GetCurrentDatabase()
+		}
+		if dbName == "" {
+			return nil, sql.ErrNoDatabaseSelected.New()
+		}
 		return plan.NewShowCreateProcedure(
 			sql.UnresolvedDatabase(s.Table.Qualifier.String()),
```

Comment:
Looks like this probably needs to be updated to `dbName`

Reply:
Yup, I must have merged a wrong commit, fixing.